pattern-library/sass/patterns/_forms.scss

```diff
@@ -257,6 +257,14 @@ $input-width: (
             border-color: palette(error, text);
         }
     }
+
+    // Input fields immediately followed by a button should eliminate the spacing between them
```

Comment:
Mmm. Good thought. We don't currently have such a class - I could set it up this way, though:

```css
input[type=search] .form-input {
  & + button {
  }
}
```

so that it only applies to search boxes?